openerp/tools/translate.py

```diff
@@ -742,18 +742,20 @@ def get_root_view(xml_id):
                 _logger.error("name error in %s: %s", xml_name, str(exc))
                 continue
             objmodel = registry.get(obj.model)
-            if (objmodel is None or field_name not in objmodel._columns
+            if (objmodel is None or (
+                        field_name not in objmodel._columns and
+                        field_name not in objmodel._fields)
```

Comment:
I meant the other way around. I know there are fields not present in columns, such as computed non stored fields, for example. But are there any columns not present in fields?